build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'com.android.library'
+
+dependencies {
+    compile 'com.android.support:support-v4:21.0.3'
```

Comment:
Agreed, I am so used to these support libraries that I instinctively include them. Fixed in the latest commit :)